### PR TITLE
Enhance: Team page

### PR DESCRIPTION
**Pull request by BobbyLeeSH**
- Not yet tested on QA
- Need filter out logic to be verified
  - eg. what if he actually played but made no stats at all?

### CODE REVIEW

**Comment by taemin410** [comment]
> - Not yet tested on QA
> 
> - Need filter out logic to be verified
> 
>   - eg. what if he actually played but made no stats at all?

I think that would at least have entry with zeros. (as opposed to no row in sql at all)

File: src/components/Player/PlayerMatchStats.js [hunk 1]
```diff
@@ -1,11 +1,42 @@
...
+              stat.PF != 0 ||
+              stat.EFF != 0 ||
+              stat.TSP != 0 ||
+              stat["FG%"] != 0 ||
+              stat["3P%"] != 0 ||
+              stat["FT%"] != 0
+          )
+        : []
+    );
+  }, [playerMatchStats]);
```
Comment (taemin410) [inline src/components/Player/PlayerMatchStats.js:39]: 여기가 어떤 역할이죵?

Comment (BobbyLeeSH) [reply src/components/Player/PlayerMatchStats.js:39]: 아 이것도 티켓 잘 못 이해해서 넣어둔 부분이었는데 날리겠슴다!

**Review by taemin410 — APPROVED** [review]
Lgtm

**Pull request merged**